### PR TITLE
Merge back Release v91.1.1 and v91.1.2 back into main

### PR DESCRIPTION
**Pull request by skhamis**
This merges the fixes in v91 back into main

### Pull Request checklist ###

- [ ] **Quality**: This PR builds and tests run cleanly
  - Note:
    - For changes that need extra cross-platform testing, consider adding `[ci full]` to the PR title.
    - If this pull request includes a breaking change, consider [cutting a new release](https://github.com/mozilla/application-services/blob/main/docs/howtos/cut-a-new-release.md) after merging.
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes a changelog entry in [CHANGES_UNRELEASED.md](../CHANGES_UNRELEASED.md) or an explanation of why it does not need one
  - Any breaking changes to Swift or Kotlin binding APIs are noted explicitly
- [ ] **Dependencies**: This PR follows our [dependency management guidelines](https://github.com/mozilla/application-services/blob/main/docs/dependency-management.md)
  - Any new dependencies are accompanied by a summary of the due dilligence applied in selecting them.

### CODE REVIEW

**Review by mhammond — APPROVED** [review]

File: .buildconfig-android.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-libraryVersion: 91.1.0
+libraryVersion: 91.1.2
```
Comment (bendk) [inline .buildconfig-android.yml:1]: I think this needs a rebase, since main is on `93.0.0`

File: components/autofill/src/db/schema.rs [hunk 2]
```diff
@@ -82,6 +82,17 @@ impl ConnectionInitializer for AutofillConnectionInitializer {
...
+        let initial_pragmas = "
+            -- use in-memory storage
+            PRAGMA temp_store = 2;
+            -- use write-ahead logging
+            PRAGMA journal_mode = WAL;
+            -- autofill does not use foreign keys at present but this is probably a good pragma to set
+            PRAGMA foreign_keys = ON;
+        ";
+        conn.execute_batch(initial_pragmas)?;
+
```
Comment (bendk) [inline components/autofill/src/db/schema.rs:95]: ..and I think this has already been merged in to main.

**Review by bendk — COMMENTED** [review]
Looks good to me, but could you rebase it to the latest main and double check that my comments are accurate?  I just want to make sure that I have a clear picture of what's getting newly added and what's already been added.

**Comment by skhamis** [comment]
Merging problems here with a protected branch is rough to get around so I'll close this in favor of this: https://github.com/mozilla/application-services/pull/4917 (Which has ben's requested changes thanks to a proper rebase)

**Pull request closed without merging**